### ISSUE TITLE
Migrate 'coreos/fedora-coreos-tracker' to 'Webhook To Fedora Messaging'

### ISSUE THREAD

**Issue by gridhead** [coreos/fedora-coreos-tracker#1826]
This project was listed in the [**GitHub2FedMsg**](https://github.com/fedora-infra/github2fedmsg) database and we want to you to inform you about the upcoming deprecation of the service. As the [**Fedora Infrastructure**](https://docs.fedoraproject.org/en-US/infra/) is finishing up with migrating its applications away from [**FedMsg**](https://github.com/fedora-infra/fedmsg) to [**Fedora Messaging**](https://github.com/fedora-infra/fedora-messaging), we encourage you to migrate your repository to the successor of the [**GitHub2FedMsg**](https://fedora-arc.readthedocs.io/en/latest/github2fedmsg/index.html) project, [**Webhook To Fedora Messaging**](https://webhook2fedmsg.apps.ocp.fedoraproject.org/docs).

Please follow [this link](https://communityblog.fedoraproject.org/announcing-webhook-to-fedora-messaging/) to the official announcement of the project’s release and use the instructions there to [migrate](https://pagure.io/fedora-infra/w2fm-migration/issues) to the new service. If this notification was a mistake, please close this notification ticket. We will not act on the repositories whose migrations have not been requested and any related GitHub2FedMsg operations will stop working once the service is decommissioned.

---

- https://github.com/coreos/afterburn/issues/1127
- https://github.com/coreos/bootupd/issues/763
- https://github.com/coreos/butane/issues/551
- https://github.com/coreos/console-login-helper-messages/issues/124
- https://github.com/coreos/coreos-installer/issues/1546
- https://github.com/coreos/fedora-coreos-cincinnati/issues/100
- https://github.com/coreos/fedora-coreos-config/issues/3242
- https://github.com/coreos/fedora-coreos-docs/issues/678
- https://github.com/coreos/ignition/issues/1965
- https://github.com/coreos/zincati/issues/1232
- https://github.com/openshift/os/issues/1647
- https://github.com/ostreedev/ostree/issues/3335

**Comment by jlebon** [comment]
I don't think we need to migrate any of those. We don't really need GitHub notifications on the bus for any of those repos.

The only repo we monitor on the bus AFAIK is https://github.com/coreos/fedora-coreos-config, so that [coreos-koji-tagger](https://github.com/coreos/fedora-coreos-releng-automation/tree/main/coreos-koji-tagger) knows when there are updated lockfiles to process. But that repo is presumably already migrated since it's not on the list (and coreos-koji-tagger definitely is using fedora-messaging).

**Comment by jlebon** [comment]
Oh wait, I see you filed https://github.com/coreos/fedora-coreos-config/issues/3242.

So basically we need to file an issue against https://pagure.io/fedora-infra/w2fm-migration/issues and that's it? Do consuming apps need to be adapted in any way or it'll just keep working exactly the same?

**Comment by gridhead** [comment]
@jlebon yes, and we should take it from there. The consuming apps should be able to get messages from the Fedora Messaging bus so as long as they are listening to it, things should work the way they are.

**Comment by dustymabe** [comment]
> I don't think we need to migrate any of those. We don't really need GitHub notifications on the bus for any of those repos.

but isn't this what is behind the glue for sync2jira syncing?

**Comment by gridhead** [comment]
@jlebon Bundle em up in one ticket if you wish to move the entire namespace over to W2FM.

**Comment by travier** [comment]
@gridhead Have you reached out to the developers of Sync2Jira? https://github.com/release-engineering/Sync2Jira

**Comment by dustymabe** [comment]
> @gridhead Have you reached out to the developers of Sync2Jira? https://github.com/release-engineering/Sync2Jira

Basically a lot of consumers of github2fedmsg are probably using it explicitly for syncing issues into Jira via sync2jira. Those users won't be able to migrate until sync2jira supports it. We are wondering if you've discussed them with that team.

**Comment by dustymabe** [comment]
> I see that @ralphbean has made a bulk request against the migration issue tracker

That request is private I assume?

**Comment by dustymabe** [comment]
The [github2fedmsg decommissioning](https://lists.fedoraproject.org/archives/list/infrastructure@lists.fedoraproject.org/message/UGMBWWAXIM6BHK3RIBWQWCNIUYZ6U5XH/) is coming up soon.

**Comment by jlebon** [comment]
I think we still need to update the webhooks for our repos.

And... I don't see where that's documented. What's the endpoint we should point to? And the shared secret? Do you have self-service step-by-step docs somewhere?

**Comment by abompard** [comment]
@ralphbean has the endpoint and shared secret, but if you open a ticket in [our private tracker](https://pagure.io/fedora-infra/w2fm-migration/issues) I'll give them to you directly too.

**Comment by ralphbean** [comment]
Apologies for the silence on this. I'm working with Dusty now to get it set up.

**Comment by dustymabe** [comment]
ok. one thing that is new is that the webhooks are enabled at the org level and not individual repo level.

For now I've enabled webhooks for `coreos` and `ostreedev` orgs. I'll work to get it enabled fpr the `openshift` org too.

**Comment by dustymabe** [comment]
openshift/os has been updated now.
